src/main/java/org/nebula/contrib/ngbatis/config/NebulaNgbatisConfig.java

```diff
@@ -0,0 +1,49 @@
+package org.nebula.contrib.ngbatis.config;
```

Comment:
Could you help me to add the license for new file.

Reply:
ok